components/courses/app/views/courses/seasons/show.slim

```diff
@@ -0,0 +1,14 @@
+h3 = season.title
+p = season_dates
+h5 = season.status
+p = season.description
+
+br
+
+h2 = t 'mentors.plural'
+- mentors.each do |mentor|
+  p = mentor.full_name
+
+br
+
+p = season_links
```

Comment:
and here you can only mention all links that could be available for the page Allow logic inside helper method decide to show itself or not.